frameworks/remix-run/application/src/routes/$langstore.order.$id.tsx

```diff
@@ -26,6 +26,9 @@ export const loader: LoaderFunction = async ({ request, params }) => {
 };
 
 export default () => {
-    const { orderId, cartId } = useLoaderData();
+    const { orderId, cartId } = useLoaderData() as {
```

Comment:
`useLoaderData<typeof loader>();` + change the loader using `LoaderFunctionArgs`